LICENSE-MIT.txt

```diff
@@ -0,0 +1,395 @@
+Attribution 4.0 International
```

Comment:
We are not using MIT LICENSE, hence we should file name from `LICENSE-MIT.txt`  to LICENSE.txt or similar.
@Ayush9026 WYT?